corehq/motech/fhir/templates/fhir/authorize.html

```diff
@@ -0,0 +1,46 @@
+{% extends "oauth2_provider/base.html" %}
+
+{% load i18n %}
+{% block content %}
+    <div class="block-center">
+        {% if not error %}
+            <form id="authorizationForm" method="post">
+                <h3 class="block-center-heading">{% trans "Authorize" %} {{ application.name }}?</h3>
```

Comment:
Word order and punctuation might depend on the language. `blocktrans` would be safer.
```suggestion
                <h3 class="block-center-heading">{% blocktrans %}Authorize {{ application.name }}?{% endblocktrans %}</h3>
```